### PR TITLE
changing default model to instructlab model

### PR DESCRIPTION
**Pull request by mairin**
Using instructlab/merlinite-7b-lab

I am making this change so we are using an InstructLab community model by default with the instructlab cli.

### CODE REVIEW

**Comment by mairin** [comment]
i still need to test this.

File: README.md [hunk 1]
```diff
@@ -199,7 +199,7 @@ The full process is described graphically in the [workflow diagram](./docs/workf
 
    ```shell
    (venv) $ ilab serve
-   INFO 2024-03-02 02:21:11,352 lab.py:201 Using model 'models/ggml-merlinite-7b-0302-Q4_K_M.gguf' with -1 gpu-layers and 4096 max context size.
+   INFO 2024-03-02 02:21:11,352 lab.py:201 Using model 'models/ggml-merlinite-7b-lab-0302-Q4_K_M.gguf' with -1 gpu-layers and 4096 max context size.
```
Comment (xukai92) [inline README.md:202]: i guess it should be no longer labelled with `0302`?

**Comment by mairin** [comment]
Tested with `ilab download` and it's working fine

**Review by xukai92 — APPROVED** [review]
also test `ilab train`, `ilab test` and `ilab convert` followed by `ilab serve` of the new model. all good.

**Pull request merged**